src/storage-controller/src/lib.rs

```diff
@@ -931,13 +995,20 @@ where
                         | IntrospectionType::SessionHistory
                         | IntrospectionType::StatementLifecycleHistory
                         | IntrospectionType::SqlText => {
-                            // do nothing.
+                            // NOTE(aljoscha): We never remove from these
+                            // collections. Someone, at some point needs to
+                            // think about that!
+                            self.collection_manager.register_append_only_collection(id);
```

Comment:
You could link the issue here, to ensure this comment doesn't become stale.

Reply:
Will do!